model/user/db.go

```diff
@@ -224,6 +224,21 @@ func FindBySlackUsername(ctx context.Context, userName string) (*DBUser, error)
 	return &u, nil
 }
 
+func FindByEmail(ctx context.Context, email string) (*DBUser, error) {
```

Comment:
this still needs a function comment (public functions always need a function comment)